lib/nerves_network/eapol_manager.ex

```diff
@@ -197,7 +211,14 @@ defmodule Nerves.Network.EAPoLManager do
       port = start_supplicant_port(state)
 
       # give it time to open the pipe
-      :timer.sleep(250)
+      wpa_control_pipe(state)
+      |> wait_for_file(111, 3_333)
+      |> case do
+        :ok ->
+          Logger.debug("Control pipe exists. Starting wpa_ex...")
+        :timeout ->
+          Logger.warn("WPA control pipe #{inspect wpa_control_pipe(state)} not created on timee!")
```

Comment:
timee!
also maybe an error instead of a warn ?

Reply:
corrected the typo. I do not mind changing to error. I think warn is more appropriate. it will continue attempting to open  the control pipe. It is also unlikely to take longer than 1[s] and even less likely to take longer 3.3[s].